### PR TITLE
chore(android): How to define in-app frames for obfuscated builds

### PR DESCRIPTION
**Pull request by romtsn**

### CODE REVIEW

File: src/platforms/android/common/proguard.mdx [hunk 1]
```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
...
+
+## In-App Frames
+
+For unobfuscated builds, the Android SDK will automatically set all frames coming from the App as `in-app` frames, based on `applicationId/packageName`. However, for obfuscated builds this is not possible on the client side. Therefore, you can define server-side rules for identifying the in-app frames, for example:
+
+```discover {tabTitle:Enhancement Rules}
+stack.module:com.mycompany.** +app
+```
+
+For more information see [StackTrace Rules](/product/data-mana/gement-settings/event-grouping/stack-trace-rules/#mark-in-app-frames/)
```
Comment (romtsn) [inline src/platforms/android/common/proguard.mdx:25]:
```suggestion
For more information see [StackTrace Rules](/product/data-mana/gement-settings/event-grouping/stack-trace-rules/#mark-in-app-frames/).
```

File: src/platforms/android/common/proguard.mdx [hunk 2]
```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
...
+
+## In-App Frames
+
+For unobfuscated builds, the Android SDK will automatically set all frames coming from the App as `in-app` frames, based on `applicationId/packageName`. However, for obfuscated builds this is not possible on the client side. Therefore, you can define server-side rules for identifying the in-app frames, for example:
+
+```discover {tabTitle:Enhancement Rules}
+stack.module:com.mycompany.** +app
+```
+
+For more information see [StackTrace Rules](/product/data-mana/gement-settings/event-grouping/stack-trace-rules/#mark-in-app-frames/).
```
Comment (romtsn) [inline src/platforms/android/common/proguard.mdx:25]:
```suggestion
For more information see [StackTrace Rules](/product/data-management-settings/event-grouping/stack-trace-rules/#mark-in-app-frames).
```

File: src/platforms/android/common/proguard.mdx [hunk 3]
```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
 The Android SDK ships with ProGuard rules automatically defined; no further configuration is needed.
 
 </Note>
+
+## In-App Frames
```
Comment (lizokm) [inline src/platforms/android/common/proguard.mdx:17]:
```suggestion
## In-app Frames
```

File: src/platforms/android/common/proguard.mdx [hunk 4]
```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
...
+
+For unobfuscated builds, the Android SDK will automatically set all frames coming from the App as `in-app` frames, based on `applicationId/packageName`. However, for obfuscated builds this is not possible on the client side. Therefore, you can define server-side rules for identifying the in-app frames, for example:
+
```
Comment (lizokm) [inline src/platforms/android/common/proguard.mdx:20]:
```suggestion
If you have an deobfuscated build, the Android SDK will automatically set all frames coming from the app as in-app, based on the `applicationId/packageName`. This is not possible on the client side for obfuscated builds. But you can define server-side rules for identifying the in-app frames, for example:

```

**Review by lizokm — APPROVED** [review]
Made some tweaks to make it flow a little bit more smoothly. Please let me know if I accidentally changed the meaning of anything.

File: src/platforms/android/common/proguard.mdx [hunk 5]
```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
 The Android SDK ships with ProGuard rules automatically defined; no further configuration is needed.
 
 </Note>
+
+## In-app Frames
+
+If you have an deobfuscated build, the Android SDK will automatically set all frames coming from the app as in-app, based on the `applicationId/packageName`. This is not possible on the client side for obfuscated builds. But you can define server-side rules for identifying the in-app frames, for example:
```
Comment (imatwawana) [inline src/platforms/android/common/proguard.mdx:19]: Small wording tweak. Also we might need to hyphenate deobfuscate, but will circle back on that.
```suggestion
If you have an deobfuscated build, the Android SDK will automatically set all frames coming from the app as in-app, based on the `applicationId/packageName`. This is not possible on the client side for obfuscated builds. However, you can define server-side rules for identifying the in-app frames, for example:
```

**Pull request merged**